package.json

```diff
@@ -12,6 +12,9 @@
     "react-redux": "^7.2.6",
     "react-router-dom": "^6.2.1",
     "react-scripts": "5.0.0",
+    "redux": "^4.1.2",
```

Comment:
- [ ] It would be better if we make our PR professional by editing the PR and providing more clarity; we can talk about the redux store, reducer, and components. This will give us a clear impression of the project hence making it more professional